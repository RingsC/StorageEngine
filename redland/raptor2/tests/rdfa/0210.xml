<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>Test 0210</title>
    <base href="http://example.org/"/>
  </head>
  <body>
    <!-- prefix has a higher priority than xmlns, but should be ignored because version is explicitly set to 1.0 -->
  <div about ="#me" xmlns:foaf="http://www.example.com/wrong/foaf/uri/" prefix="foaf: http://xmlns.com/foaf/0.1/" >
      <p property="foaf:name">Ivan Herman</p>
  </div>
  </body>
</html>