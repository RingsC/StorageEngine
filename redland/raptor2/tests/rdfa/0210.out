<http://example.org/#me> <http://www.example.com/wrong/foaf/uri/name> "Ivan Herman" .
